config/routes.rb

```diff
@@ -102,7 +102,7 @@
       get :unpublish
       get :destroy
       post 'update_featured_pages', :on => :collection
-      patch :preview
+      get :preview
```

Comment:
The method here I think had to do with browsers and webservers having various limits on how long a url (including the query string) can be. [Fastly](https://docs.fastly.com/guides/debugging/resource-limits#request-and-header-limits) for example will send back an error if the url exceeds 8KB. The preview I just checked requested a 5KB long url, which is getting up there, so it'd be safer to leave this as a post/patch so the form data is sent in the request body instead.